<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FlyWork - Авторизация</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/images.qrc">
    <normaloff>:/authdialog/images/flywork_client.png</normaloff>:/authdialog/images/flywork_client.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#AuthDialog {
	background-image: url(:/authdialog/images/authdialog_bg.png);
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="AuthLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lName">
      <property name="text">
       <string>Логин:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leName">
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>Введите Ваш логин</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lPassword">
      <property name="text">
       <string>Пароль:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lePassword">
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Введите Ваш пароль</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="pbAuth">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Авторизация</string>
      </property>
      <property name="icon">
       <iconset resource="../../../resources/images.qrc">
        <normaloff>:/authdialog/images/pbAuth_icon.png</normaloff>:/authdialog/images/pbAuth_icon.png</iconset>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>183</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #888;</string>
   </property>
   <property name="text">
    <string>Авторизируйтесь в системе FlyWork</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
